dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -190,4 +216,1141 @@ public void findAllRelationshipTest() throws Exception {
+                                          .withName("Parent Community")
+                                          .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                                           .withName("Sub Community")
+                                           .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
+        Collection col3 = CollectionBuilder.createCollection(context, child1).withName("OrgUnits").build();
+
+        Item author1 = ItemBuilder.createItem(context, col1)
+                                 .withTitle("Author1")
+                                 .withIssueDate("2017-10-17")
+                                 .withAuthor("Smith, Donald")
+                                 .withRelationshipType("Person")
+                                 .build();
+
+        Item publication = ItemBuilder.createItem(context, col3)
+                                      .withTitle("Publication1")
+                                      .withAuthor("Testy, TEst")
+                                      .withIssueDate("2015-01-01")
+                                      .withRelationshipType("Publication")
+                                      .build();
+
+        RelationshipType isAuthorOfPublicationRelationshipType = relationshipTypeService
+            .findbyTypesAndLabels(context, entityTypeService.findByEntityType(context, "Publication"),
+                                  entityTypeService.findByEntityType(context, "Person"),
+                                  "isAuthorOfPublication", "isPublicationOfAuthor");
+
+
+
+        EPerson user = ePersonService.create(context);
+        user.setFirstName(context, "first");
+        user.setLastName(context, "last");
+        user.setEmail("testaze@email.com");
+        user.setCanLogIn(true);
+        user.setLanguage(context, I18nUtil.getDefaultLocale().getLanguage());
+        ePersonService.setPassword(user, password);
+        // actually save the eperson to unit testing DB
+        ePersonService.update(context, user);
+        context.setCurrentUser(user);
+
+        authorizeService.addPolicy(context, publication, Constants.WRITE, user);
+
+        String token = getAuthToken(user.getEmail(), password);
+
+        MvcResult mvcResult = getClient(token).perform(post("/api/core/relationships")
+                                                           .param("relationshipType",
+                                                                  isAuthorOfPublicationRelationshipType.getID()
+                                                                                                       .toString())
+                                                           .contentType(MediaType.parseMediaType("text/uri-list"))
```

Comment:
All of the hardcoded `text/uri-list` strings in your Tests should likely be replaced with `org.springframework.data.rest.webmvc.RestMediaTypes.TEXT_URI_LIST_VALUE`   We don't need to be hardcoding this over and over again, just like we don't hardcode `application/json` in our tests.